build.gradle

```diff
@@ -8,8 +8,7 @@ buildscript {
     }
     dependencies {
         // wasn't working with travis
-//        classpath 'com.android.tools.build:gradle:3.0.0-beta7'
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.0'
```

Comment:
This will prob make CI fail until we get the BridgeAndroidSdk submodule updated which I think Josh has up for review.